README.md

```diff
@@ -1,202 +1,85 @@
 # dbt Wizard
 
-This extension will help you work with [dbt](https://www.getdbt.com/).
-Also, it provides additional features like [Highlighting errors](#highlighting-errors), [Completion for SQL](#completion-for-sql), etc., for BigQuery data warehouses. 
+Working with [dbt Core](https://www.getdbt.com/) in Visual Studio Code using the Fivetran dbt Wizard extension accelerates your first-time environment setup with dbt Core, and optimizes your continual development of transformation pipelines. This extension is designed primarily for BigQuery destinations, but support for other warehouses is in development.
```

Comment:
"support for other warehouses" is already partially implemented (completions, definitions). Do we need to designate this?